tests/js/HappinessLintBearTest.py

```diff
@@ -14,6 +23,32 @@
 var x=2
 """
 
+good_file1 = """
+if (options.quiet !== true)
+  console.log('done')
+var x = 2
+"""
+
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    USE_SPACE_ERR_MSG = re.escape('"use_spaces=True" is incompatible with '
+                                  'HappinessLintBear, set it to false.')
+
+    def test_validate_use_spaces(self):
+        section = Section('name')
+        section.append(Setting('use_spaces', True))
+        bear = HappinessLintBear(section, Queue())
+
+        with self.assertRaisesRegex(AssertionError, self.USE_SPACE_ERR_MSG):
+            content = bad_file.splitlines()
+            with prepare_file(content, None) as (file, fname):
+                with execute_bear(bear, fname, file) as results:
+                    fdict = {fname: file}
+                    results[0].apply(fdict)
+                    self.assertEqual(''.join(fdict[fname]), good_file1)
```

Comment:
-->
```
with self.assertRaisesRegex(ValueError, ...)
    bear.execute('test_filename', [])
```

Then you can remove `good_file1` and a few other imports

Reply:
I dont think there are any extra imports coz otherwise the ci ould have throw error.